bears/general/InvalidLinkBear.py

```diff
@@ -68,13 +68,49 @@ def find_links_in_file(file, timeout, link_ignore_regex):
                                         # set of parenthesis.
                                         # An example can be:
                                         # http://wik.org/Hello_(Adele_song)/200
-            )
-            *)
-                                        # Thus, the whole part above
+            )                           # Thus, the whole part above
                                         # prevents matching of
                                         # Unbalanced parenthesis
+            (
+                (%?[0-9]+)              # A ``%`` sign (0 or 1 times)
+                                        # followed by any number
+                                        # Restricts URLs to end with
+                                        # a number (HTTP Status Code)
+                [^\s()%\'"`<>|\\]+      # Same Regex for Path Name
```

Comment:
Regex and Path Name should be all lower case (here and elsewhere, only use capitals for start of sentence , or proper nouns , or abbreviations).